# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# T2 SDE: package/.../sysvinit/dev_vc_0.patch
# Copyright (C) 2004 - 2005 The T2 SDE Project
# Copyright (C) 1998 - 2003 ROCK Linux Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---

diff -Nur sysvinit-2.85.orig/src/bootlogd.c sysvinit-2.85/src/bootlogd.c
--- sysvinit-2.85.orig/src/bootlogd.c	2001-12-09 15:01:28.000000000 +0100
+++ sysvinit-2.85/src/bootlogd.c	2003-08-30 15:49:12.000000000 +0200
@@ -318,7 +318,7 @@
 	(void)ioctl(0, TIOCCONS, NULL);
 #if 1
 	/* Work around bug in 2.1/2.2 kernels. Fixed in 2.2.13 and 2.3.18 */
-	if ((n = open("/dev/tty0", O_RDWR)) >= 0) {
+	if ((n = open("/dev/vc/0", O_RDWR)) >= 0) {
 		(void)ioctl(n, TIOCCONS, NULL);
 		close(n);
 	}
diff -Nur sysvinit-2.85.orig/src/paths.h sysvinit-2.85/src/paths.h
--- sysvinit-2.85.orig/src/paths.h	2003-04-14 13:37:01.000000000 +0200
+++ sysvinit-2.85/src/paths.h	2003-08-30 15:49:38.000000000 +0200
@@ -13,7 +13,7 @@
  *		as published by the Free Software Foundation; either version
  *		2 of the License, or (at your option) any later version.
  */
-#define VT_MASTER	"/dev/tty0"		/* Virtual console master */
+#define VT_MASTER	"/dev/vc/0"		/* Virtual console master */
 #define CONSOLE		"/dev/console"		/* Logical system console */
 #define SECURETTY	"/etc/securetty"	/* List of root terminals */
 #define SDALLOW		"/etc/shutdown.allow"	/* Users allowed to shutdown */
